internal/controllers/validators/validators_test.go

```diff
@@ -1,63 +1,94 @@
 package validators_test
 
 import (
-	. "github.com/onsi/ginkgo/v2"
-	. "github.com/onsi/gomega"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
 
 	"github.com/operator-framework/operator-controller/api/v1alpha1"
 	"github.com/operator-framework/operator-controller/internal/controllers/validators"
 )
 
-var _ = Describe("Validators", func() {
-	Describe("ValidateOperatorSpec", func() {
-		It("should not return an error for valid SemVer", func() {
-			operator := &v1alpha1.Operator{
-				Spec: v1alpha1.OperatorSpec{
-					Version: "1.2.3",
-				},
-			}
-			err := validators.ValidateOperatorSpec(operator)
-			Expect(err).NotTo(HaveOccurred())
-		})
-
-		It("should return an error for invalid SemVer", func() {
-			operator := &v1alpha1.Operator{
-				Spec: v1alpha1.OperatorSpec{
-					Version: "invalid-semver",
-				},
-			}
-			err := validators.ValidateOperatorSpec(operator)
-			Expect(err).To(HaveOccurred())
-		})
-
-		It("should not return an error for empty SemVer", func() {
-			operator := &v1alpha1.Operator{
-				Spec: v1alpha1.OperatorSpec{
-					Version: "",
-				},
-			}
-			err := validators.ValidateOperatorSpec(operator)
-			Expect(err).NotTo(HaveOccurred())
-		})
+type testData struct {
```

Comment:
Scoping comments for this struct and data apply here (why shouldn't another test for another behavior in this package have their own `testData`?)

Reply:
This is literally a single-file test in this single-file package... I was having trouble making an anonymous struct, and gave up on it... I should go back and look.